src/handlers/user/loginHandler.ts

```diff
@@ -1,6 +1,22 @@
-const loginHandler = (
+const {User} = require('../../db')
+const bcrypt = require('bcrypt')
+
+
+const loginHandler = async (
   email: string | undefined,
   password: string | undefined
-) => {};
+) => {
+  if(!email || !password) throw new Error ('Missing data.')
+
+  const user = await User.findOne({where: email})
+
+  if(!user) throw new Error ("User doesn't exists...")
+
+  const isCorrect = await bcrypt.compare(password, user.password);
+
+  if(isCorrect) return user;
```

Comment:
## QoL Change
As suggested by you in the PR, we could return an object containing a boolean variable instead of the user information for easier verification inside our Frontend:
Suggested format:
```javascript
if (isCorrect) return { auth: isCorrect }
```